packages/amplify_core/lib/src/config/api/appsync/api_config.dart

```diff
@@ -22,7 +22,7 @@ class AWSApiConfig with AWSEquatable<AWSApiConfig>, AWSSerializable {
   final EndpointType endpointType;
   final String endpoint;
   final String region;
-  final APIAuthorizationType authorizationType;
+  final APIAuthorizationMode authorizationType;
```

Comment:
Oh this is why it was called authorization type - I think we should just align with what other SDKs are doing

Reply:
discussed offline but changed it back